source/documentation/requirements.html.md

```diff
@@ -53,15 +55,15 @@ documentation](https://github.com/openstack/requirements/#proposing-changes)
 
     <br />
 2. Check if the new requirement is present in CentOS base channels. The easiest way to do this
-is using yum command from a system running CentOS 7:
+is using 'repoquery' command from a system running RPM based OS(CentOS/RHEL/Fedora):
 
-        yum list "*<dependency>"
+        repoquery --repofrompath=tmp,http://mirror.centos.org/centos/7/os/x86_64/ --repofrompath=tmp2,http://mirror.centos.org/centos/7/updates/x86_64/ --repofrompath=tmp3,http://mirror.centos.org/centos/7/extras/x86_64/ --disablerepo=* --enablerepo=tmp* "*<dependency>"
```

Comment:
Instead of a long line here I'd say it's better to use multiline:

repoquery --repofrompath=tmp,http://mirror.centos.org/centos/7/os/x86_64/ \
--repofrompath=tmp2,http://mirror.centos.org/centos/7/updates/x86_64/ \
--repofrompath=tmp3,http://mirror.centos.org/centos/7/extras/x86_64/ \
--disablerepo=* --enablerepo=tmp* "*<dependency>"